lib/Service/OpenAiSettingsService.php

```diff
@@ -461,6 +470,17 @@ public function setRequestTimeout(int $requestTimeout): void {
 		$this->appConfig->setValueString(Application::APP_ID, 'request_timeout', strval($requestTimeout));
 	}
 
+	/**
+	 * Setter for chunkSize; default/minimum is 0 (no chunking)
+	 * @param int $chunkSize
+	 * @return void
+	 */
+	public function setChunkSize(int $chunkSize): void {
+		// Validate input:
+		$chunkSize = max(0, $chunkSize);
+		$this->appConfig->setValueString(Application::APP_ID, 'chunk_size', strval($chunkSize));
```

Comment:
We could use setValueInt et getValueInt for the chunk size. I was just lazy when switching from IConfig to IAppConfig to use the type-specific methods and migrate the existing values. For a new one like chunk_size, we can go for the typed stuff IMO. (Optional change)

Reply:
It works as long as the chunk_size string hasn't been written to the database yet. Trying to switch from string to int would result in a `AppConfigTypeConflictException: conflict with value type from database`.